packages/picker/src/Picker.ts

```diff
@@ -417,6 +424,13 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
         `;
     }
 
+    protected get renderDescriptionSlot(): TemplateResult {
+        return html`
+            <div id="sp-picker">
```

Comment:
I think we should use `description` or similar here, so that when it is leveraged in Action Menu or Split Button, no one is surprised.

Reply:
Done!